birdhouse/config/jupyterhub/jupyterhub_config.py.template

```diff
@@ -62,6 +62,9 @@ def create_dir_hook(spawner):
 
 c.Spawner.pre_spawn_hook = create_dir_hook
 
+## Disable per-user configuration of single-user servers.
+c.Spawner.disable_user_config = True
```

Comment:
Not sure, but I think this has no effets when using dockerspawner?

Reply:
Apparently user can influence how the jupyter user server starts with stuff in HOME/.jupyter, and we persist HOME now ... so I'd rather be on the cautious side.  But agree by using dockerspawner, the risk is already reduced or this setting might have no effect.  Was following the doc in the commit message.